workspaces/sonarqube/.changeset/sixty-tables-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-sonarqube': minor
```

Comment:
```suggestion
'@backstage-community/plugin-sonarqube': patch
```
I think patch makes more sense since it's just a doc update. WDYT?

Reply:
Sounds good to me! Just approved the suggestion